### ISSUE TITLE
Error: blackfire repo changed

### ISSUE THREAD

**Issue by Paviluf** [laravel/homestead#1009]
Hello,
There is an error on first vagrant up since the blackfire repo have changed :

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

apt-get update

Stdout from the command:

Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [83.2 kB]
...
Get:41 https://packages.microsoft.com/ubuntu/18.04/prod bionic/main amd64 Packages [32.0 kB]
Reading package lists...

Stderr from the command:

E: Repository 'http://packages.blackfire.io/debian any InRelease' changed its 'Origin' value from 'packagecloud.io/sensiolabs/sensiolabs' to 'packagecloud.io/blackfire-io/stable'
E: Repository 'http://packages.blackfire.io/debian any InRelease' changed its 'Label' value from 'sensiolabs' to 'stable'
```

Thanks !

**Comment by DmitryNiko** [comment]
Hi, i have the same problem.

**Comment by svpernova09** [comment]
Run this from inside the VM: `sudo apt update` and answer `y` to the question:

```
root@homestead:/home/vagrant# sudo apt update
Get:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [83.2 kB]
Hit:2 https://packages.microsoft.com/ubuntu/18.04/prod bionic InRelease
Hit:4 https://deb.nodesource.com/node_8.x bionic InRelease
Hit:3 https://cli-assets.heroku.com/apt ./ InRelease
Hit:5 http://archive.ubuntu.com/ubuntu bionic InRelease
Hit:6 http://ppa.launchpad.net/nginx/development/ubuntu bionic InRelease
Hit:7 http://archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:8 https://dist.crystal-lang.org/apt crystal InRelease
Hit:9 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease
Hit:10 http://archive.ubuntu.com/ubuntu bionic-backports InRelease
Get:11 http://packages.blackfire.io/debian any InRelease [23.2 kB]
E: Repository 'http://packages.blackfire.io/debian any InRelease' changed its 'Origin' value from 'packagecloud.io/sensiolabs/sensiolabs' to 'packagecloud.io/blackfire-io/stable'
E: Repository 'http://packages.blackfire.io/debian any InRelease' changed its 'Label' value from 'sensiolabs' to 'stable'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
Do you want to accept these changes and continue updating from this repository? [y/N] y
Get:12 http://packages.blackfire.io/debian any/main amd64 Packages [1,218 B]
Get:13 http://packages.blackfire.io/debian any/main i386 Packages [1,198 B]
Fetched 109 kB in 9s (12.4 kB/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
```

**Comment by Paviluf** [comment]
Thanks, the issue was easy to solve but I reported it for it to be fixed in the box. Will there be a new release with the "fix" ?

**Comment by browner12** [comment]
can you submit a PR?

**Comment by svpernova09** [comment]
> Thanks, the issue was easy to solve but I reported it for it to be fixed in the box. Will there be a new release with the "fix" ?

Yep!

**Comment by zitivapo** [comment]
Is there any ETA for the next box release? Thanks!

**Comment by svpernova09** [comment]
> Is there any ETA for the next box release? Thanks!

[Soon™](https://www.urbandictionary.com/define.php?term=soon#4387130)

**Comment by IllyaMoskvin** [comment]
Just for the record for anyone else who finds their way here from Google, if you see this error:
```
vagrant@aic:~$ sudo apt-get update
[...]
Get:8 http://packages.blackfire.io/debian any InRelease [23.2 kB]
Err:8 http://packages.blackfire.io/debian any InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 696DBE66A72D76DA
Fetched 275 kB in 3s (94.6 kB/s)
Reading package lists... Done
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://packages.blackfire.io/debian any InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 696DBE66A72D76DA
W: Failed to fetch http://packages.blackfire.io/debian/dists/any/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 696DBE66A72D76DA
W: Some index files failed to download. They have been ignored, or old ones used instead.
```
Running this command fixed the issue for me:
```
sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 696DBE66A72D76DA
```
After that, you will see the error described in the OP.
